### ISSUE TITLE
I can't write ; on Hungarian keyboard

### ISSUE THREAD

**Issue by Estalhun** [tomasr/viasfora#218]
Hello,

Viasfora uses the "Ctrl + Alt + ," as a hotkey.
But "AltGr + ," writes the ";" on Hungarian keyboards.
So, I cannot write ";" to the end of C++ lines.

I think Polish and other keyboards have the same issues with AltGr (Control + Alt)

Cheers,
Estal

**Comment by tomasr** [comment]
You can re-assign this option to another key combination in the Visual Studio keyboard options.

However, a bit confused... are you saying that Ctrl + Alt is equivalent to AltGr? Or am I misinterpreting something?

**Comment by Estalhun** [comment]
No, that is correct. AltGr = Alt + Ctrl. At least in this case.

2017. szept. 23. 13:40 ezt írta ("Tomas Restrepo" <notifications@github.com
>):

You can re-assign this option to another key combination in the Visual
Studio keyboard options.

However, a bit confused... are you saying that Ctrl + Alt is equivalent to
AltGr? Or am I misinterpreting something?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/tomasr/viasfora/issues/218#issuecomment-331628508>, or mute
the thread
